pkg/apis/eksctl.io/v1alpha5/defaults.go

```diff
@@ -123,3 +123,11 @@ func DefaultClusterNAT() *ClusterNAT {
 		Gateway: &single,
 	}
 }
+
+// ClusterEndpointAccessDefaults returns a ClusterEndpoints pointer with default values set.
+func ClusterEndpointAccessDefaults() *ClusterEndpoints {
+	return &ClusterEndpoints{
+		PrivateAccess: Disabled(),
```

Comment:
I think it's acceptable to change the default for new clusters and enable private access, as currently private nodegroups have to go via NAT gateway to access the API...

Reply:
Should I then give the message that they'll need to update securitygroups/etc in order for clients within the VPC to communicate with the Kubernetes API whenever we create a cluster with these defaults?

Reply:
Need to discuss the UX of this.  Seems like we need to say something about what users will need to do to make private access actually work.  I can put the AWS URL that speaks to this in the output, but seems odd that the default configuration will flag immediately put this issue in front of them.